run-pre-checks.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -exu
+
+kafka_repo_root="$(realpath "$(dirname "$0")")"
+
+cd ${kafka_repo_root}/kuttl-tests
+DS_KUDO_VERSION=${DS_KUDO_VERSION} ./run.sh
```

Comment:
This seems like a no-op? Am I missing something?
```suggestion
./run.sh
```